recipes/itk-rtk/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "2.0.1" %}
+
+{% set pyver = "cp{}{}".format(PY_VER[0], PY_VER[2]) %}
+
+{% set platform = "win_amd64" %}  # [win]
+{% set platform = "manylinux1_x86_64" %}  # [linux]
+{% set platform = "macosx_10_9_x86_64" %}  # [osx]
+
+{% set fn = "{}-{}-{}-{}m-{}.whl".format("itk_rtk", version, pyver, pyver, platform) %}
+
+package:
+  name: itk-rtk
+  version: {{ version }}
+
+source:
+  fn: {{ fn }}
+  url: https://pypi.io/packages/{{ pyver }}/i/itk_rtk/{{ fn }}
+  sha256: 5db437222cbdd0af994184a3cb8e6317eae6c76a4cb42ecd8e1cbf9ca2c41bbc  # [osx and py27]
+  sha256: 08fa2604edfe532d04fadfe26b13923c80718964fd097ba192a08e3600bfed4c  # [osx and py36]
+  sha256: d4d8f042c9bc013d5a220b7027f4850e46cbeec5a9ff0d3399ed46652ab63335  # [osx and py37]
+
+  sha256: a7b2e40f3567c1c00a821413033f5531ed4490c8dc8f34182911520f1767e0c6  # [linux and py27]
+  sha256: 1484826ae76f8b66739b44ae4f947c52262c604480f11caafdbe73760de538bc  # [linux and py36]
+  sha256: 50281724f538941b268cee949c2877642027c2517dea2983be82e1381b9356ad  # [linux and py37]
+
+  sha256: 2cdb43a3666dd6ce4a8784b1c9284e515ac112c58263548ff6fb20fe2ebbb340  # [win and py36]
+  sha256: 559f5cb5b201c63623abee074fd79fa08c47f6deba72227b86d98109cdb81b49  # [win and py37]
+
+build:
+  number: 0
+  skip: True
```

Comment:
This will not create any packages because you are always skipping.  This line should probably be removed.